### PR TITLE
M sa2 10 19

### PR DESCRIPTION
**Pull request by Homln**

### CODE REVIEW

**Comment by Homln** [comment]
correct history

File: lhomitsevich/02.git.local/commit.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Homitsevich@Windows7 MINGW64 /c/Work/4/sa.it-academy.by/lhomitsevich/02.git.local (features/do_one)
```
Comment (pluhin) [inline lhomitsevich/02.git.local/commit.md]: Could you please use git markup for md files

File: lhomitsevich/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+git init   Первая команда с которой нужно начать
```
Comment (pluhin) [inline lhomitsevich/readme.md]: Could you please use git markup for md files

File: lhomitsevich/02.git.local/readme.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+git init Наша первыя команда
```
Comment (pluhin) [inline lhomitsevich/02.git.local/readme.md]: Could you please use git markup for md files

**Comment by Homln** [comment]
Fixed markup

**Pull request merged**